### DISCUSSION TITLE
My submission to my repository says I have an invalid notebook

### DISCUSSION THREAD

**Discussion by Duxst** [Part-II-Computational-Physics/.github#20] [Q&A]
I have just copied by code into the repository section and made sure to commit changes but now I have an invalid notebook.

**Comment by NathanMagnan** [comment]
This seems to be a common issue with google colab notebooks. You can find a workaround there:
https://github.com/jupyter/notebook/issues/6507

If this does not work, maybe ask yhl54 who had the same problem and found a workaround.

**Reply by AustenLamacraft** [reply]
I've just tried uploading a colab notebook to GitHub and it worked fine
